lib/shopify-cli/commands/serve.rb

```diff
@@ -6,12 +6,22 @@ module ShopifyCli
   module Commands
     class Serve < ShopifyCli::Command
       include ShopifyCli::Helpers::OS
-
-      prerequisite_task :ensure_env, :tunnel
+      prerequisite_task :ensure_env
+      options do |parser, flags|
+        parser.on('--host=HOST') do |h|
+          flags[:host] = h
+        end
+      end
 
       def call(*)
         project = Project.current
-        url = ShopifyCli::Tasks::Tunnel.call(@ctx)
+        custom_host = options.flags[:host]
+        url = if custom_host
+            custom_host
+        else
+          ShopifyCli::Tasks::Tunnel.call(@ctx)
+        end
```

Comment:
this can be simplified to `url = options.flags[:host] || Tasks::Tunnel.call(@ctx)`

Reply:
Oh yes thanks!